README.md

```diff
@@ -321,6 +321,36 @@ and add them to the media library if their MIME types are supported. If the
 file/folder in _path_ does not exist/is not readable or is not provided then an
 error will be returned and the corresponding log message would be written into logs.
 
+## Internal Audio & Video Recording
+
+REQUIRED STEPS to activate recording:
+
+```bash
+adb shell pm grant io.appium.settings android.permission.RECORD_AUDIO
+adb shell pm grant io.appium.settings android.permission.WRITE_EXTERNAL_STORAGE
+adb shell appops set io.appium.settings PROJECT_MEDIA allow
+```
+
+Start Recording:
+```bash
+adb shell am start -n "io.appium.settings/io.appium.settings.Settings" -a io.appium.settings.recording.ACTION_START --es "recording_filename" "abc.mp4" --es "recording_priority" 0 --es "recording_max_duration" 900
```

Comment:
I would rather go with simple arg names, without the `recording_` prefix. Also `max_duration_sec` would be more precise

Reply:
done https://github.com/sirmordred/io.appium.settings/commit/2c8878a8387a0b425117abfd6db12a12292b25a0